### ISSUE TITLE
Warnings in log due to incorrect use of ClimateEntityFeature entity in HA Core 2024.5.5

### ISSUE THREAD

**Issue by ngrusz1** [IngoS11/ha-schluter#73]
### System Health details

Component Version: 0.2.1
HA Core: 2024.5.5

### Checklist

- [X] I have enabled debug logging for my installation.
- [X] I have filled out the issue template to the best of my ability.
- [X] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).
- [X] This issue is not a duplicate issue of currently [previous issues](https://github.com/ludeeus/integration_blueprint/issues?q=is%3Aissue+label%3A%22Bug%22+)..

### Describe the issue

When starting Home Assistant, I get the warning noted below in my logs. The message appears related to the latest version of the Climate entity. The component needs to be updated to use the latest version of the Climate entity properly.

### Reproduction steps

1. Install the ha-schluter component.
2. Start Home Assistant.
3. Review the logs.

### Debug logs

```text
Logger: homeassistant.components.climate
Source: components/climate/__init__.py:363
integration: Climate (documentation, issues)
First occurred: 5:59:11 PM (1 occurrences)
Last logged: 5:59:11 PM

Entity None (<class 'custom_components.schluter.climate.SchluterThermostat'>) implements HVACMode(s): heat, auto, off and therefore implicitly supports the turn_on/turn_off methods without setting the proper ClimateEntityFeature. Please create a bug report at https://github.com/Ingos11/ha-schluter/issues
```

**Comment by jringuette** [comment]
This PR https://github.com/IngoS11/ha-schluter/pull/74 fix this issue

**Comment by IngoS11** [comment]
I accepted all pull requests but I did not create a new release as my own thermostat is not showing up any more in the integration.

**Comment by jringuette** [comment]
Hum, that is strange, works on my end. Any errors in the logs?

**Comment by IngoS11** [comment]
Yes, errors in the log about the config. They are coming from the non-blocking alternative that was added with #77. I removed that part again and did create v0.2.2. Seems like there is more to do if you want to change that. Unforunately I don't have enough time to follow all the changes that happen in HA to keep up with this.
